ktor-server/jvm/src/io/ktor/server/application/plugins/api/ApplicationConfigExtensions.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.application.plugins.api
+
+import io.ktor.server.config.*
+
+/**
+ * Port of the current application. Same as in config.
+ **/
+public val ApplicationConfig.port: Int get() = propertyOrNull("ktor.deployment.port")?.getString()?.toInt() ?: 8080
+
+/**
+ * Host of the current application. Same as in config.
```

Comment:
Represents the host the current application is running on, as defined by the configuration at start-up.

Reply:
Represents the host address of the currently running application, as defined by the configuration at start-up.